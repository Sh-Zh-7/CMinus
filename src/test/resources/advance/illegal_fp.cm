int main() {
    int a = 1.05e;
    int b = .e+123;
    int c = e-456;
    int d = 1fe;
    int e = 1e1.1;
    int f = 1ef;
}