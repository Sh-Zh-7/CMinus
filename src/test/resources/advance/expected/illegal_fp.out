Error type A at Line 2: Illegal floating-point numbers!
Error type A at Line 3: Illegal floating-point numbers!
Error type A at Line 4: Illegal floating-point numbers!
Error type A at Line 5: Illegal decimal numbers!
Error type A at Line 6: Illegal floating-point numbers!
Error type A at Line 7: Illegal decimal numbers!
